YML.md

```diff
@@ -1,12 +1,21 @@
 # About `.yml` files
 
-The `recipes/meta` directory in this repository contains `.yml` files.
+The `recipes/meta` directory in this repository contains `.yml` files. These are used by the `Recipe` to simplify the creation of AppImages, especially by using pre-built binaries from Debian packages, both to save time for creating and building an AppImage.
 
-## Purpose of `.yml` files 
+This document provides an introduction to the `.yml` files' purpose, their structure and a few examples that can be used with the existing `Recipe`.
+
+
+## Purpose of `.yml` files
+
+`.yml` is the file extension commonly used for _YAML_ (_Yet Another Markup Language_, nowadays also serves as an abbreviation for _YAML Ain’t Markup Language_).
+
+YAML's approach to describing data is to combine associative lists (known as `dict`s in Python or `object literal`s in JavaScript, for example), lists (arrays) and scalar values. This results in an easy to parse and also easy to read format.
+
+The `.yml` files are used by the script called `Recipe` which is used in the _AppImages_ project, more precisely the `meta` subproject, to convert binary _ingredients_ into AppImages for demonstration purposes. Their primary objective is to make it very simple to convert pre-existing binaries into the AppImage format. If you can build your software from source, you may generate AppImages directly as part of your build workflow; in this case you may not need a `.yml` file (but a Travis CI `.travis.yml` and/or a `Makefile`, etc.).
+
+The `.yml` file format is not part of the AppImage standard, which just describes the AppImage container format and is agnostic as to how the payload inside an AppImage gets generated. Neither it is part of AppImageKit, because AppImageKit is only concerned with taking a pre-existing _AppDir_ and converting that into an AppImage. Such an AppDir is created from the instructions stored in the `.yml` files, and converted to an AppImage using _AppImageKit_.
 
-`.yml` files are used by the script used in the AppImages project that converts binary ingredients into AppImages for demonstration purposes. The `.yml` file format is __not__ part of the AppImage standard which just describes the AppImage container format and is agnostic as to how the payload inside an AppImage gets generated. The `.yml` file format is also __not__ not part of AppImageKit, because AppImageKit is only concerned with taking a pre-existing AppDir and converting that into an AppImage. 
 
-The objective of `.yml` files is to make it very simple to convert pre-existing binaries into the AppImage format. If you can build your software from source, you may generate AppImages directly as part of your build workflow; in this case you may not need a `.yml` file (but a Travis CI `.travis.yml` and/or a `Makefile`, etc.
```

Comment:
I merged it with the other paragraph which said something very similar.It was a little redundant, while restructuring I moved the information to the paragraph in line 14.